src/Runtimes/Runtimes.php

```diff
@@ -20,60 +20,60 @@ public function __construct($version = '')
     {
         $this->version = $version;
 
-        $node = new Runtime('node', 'Node.js');
+        $node = new Runtime('node', 'Node.js', 'pm2 start src/server.js --no-daemon');
         $node->addVersion('14.5', 'node:14.5-alpine', 'openruntimes/node:' . $this->version . '-14.5', [System::X86, System::ARM]);
         $node->addVersion('16.0', 'node:16-alpine', 'openruntimes/node:' . $this->version . '-16.0', [System::X86, System::ARM]);
         $node->addVersion('18.0', 'node:18-alpine', 'openruntimes/node:' . $this->version . '-18.0', [System::X86, System::ARM]);
         $this->runtimes['node'] = $node;
 
-        $php = new Runtime('php', 'PHP');
+        $php = new Runtime('php', 'PHP', 'php src/server.php');
         $php->addVersion('8.0', 'php:8.0-cli-alpine', 'openruntimes/php:' . $this->version . '-8.0', [System::X86, System::ARM]);
         $php->addVersion('8.1', 'php:8.1-cli-alpine', 'openruntimes/php:' . $this->version . '-8.1', [System::X86, System::ARM]);
         $this->runtimes['php'] = $php;
 
-        $ruby = new Runtime('ruby', 'Ruby');
+        $ruby = new Runtime('ruby', 'Ruby', 'bundle exec puma -b tcp://0.0.0.0:3000 -e production');
         $ruby->addVersion('3.0', 'ruby:3.0-alpine', 'openruntimes/ruby:' . $this->version . '-3.0', [System::X86, System::ARM]);
         $ruby->addVersion('3.1', 'ruby:3.1-alpine', 'openruntimes/ruby:' . $this->version . '-3.1', [System::X86, System::ARM]);
         $this->runtimes['ruby'] = $ruby;
 
-        $python = new Runtime('python', 'Python');
+        $python = new Runtime('python', 'Python', 'python3 src/server.py');
         $python->addVersion('3.8', 'python:3.8-alpine', 'openruntimes/python:' . $this->version . '-3.8', [System::X86, System::ARM]);
         $python->addVersion('3.9', 'python:3.9-alpine', 'openruntimes/python:' . $this->version . '-3.9', [System::X86, System::ARM]);
         $python->addVersion('3.10', 'python:3.10-alpine', 'openruntimes/python:' . $this->version . '-3.10', [System::X86, System::ARM]);
         $this->runtimes['python'] = $python;
 
-        $deno = new Runtime('deno', 'Deno');
+        $deno = new Runtime('deno', 'Deno', 'denon start');
```

Comment:
typo `denon`

Reply:
I learhed this the hard way, but that seems to be actual tool 😅 https://deno.land/x/denon@2.5.0
It achieves auto-restart on crash